dlt/destinations/dataset/relation.py

```diff
@@ -202,6 +205,17 @@ def normalized_query(self) -> sge.Query:
             )
         return self._normalized_query
 
+    def __str__(self) -> str:
```

Comment:
why not just render and output the pretty formatted sql query here? or is this too abstract?

Reply:
right! we'll do this better. maybe like ibis: you show the source table, transformation and output table. this is just a quick update for 1.12.2